server/methods/core/shop.js

```diff
@@ -65,11 +60,15 @@ Meteor.methods({
       });
     }
 
-    check(shop, Schemas.Shop);
+    // We trust the owner's shop clone, check only if shopData is passed as an argument
+    if(shopData) {
+      check(shop, Schemas.Shop);
+    }
+
     try {
       Collections.Shops.insert(shop);
     } catch (error) {
-      return Logger.error("Failed to shop/createShop", sanitizedError);
+      return Logger.error("Failed to shop/createShop", error);
```

Comment:
The error object also needs to be the first arg to the logger method to be parsed correctly.